src/rrtmgp/mo_gas_optics_rrtmgp.jl

```diff
@@ -146,34 +143,39 @@ module mo_gas_optics_rrtmgp
                        optical_props::ty_optical_props_arry,
                        sources::ty_source_func_lw,
                        col_dry=nothing,
-                       tlev=nothing) # result(error_msg)
-    # inputs
-    # class(ty_gas_optics_rrtmgp), intent(in) :: this
-    # real(FT), dimension(:,:), intent(in   ) :: play,    # layer pressures [Pa, mb]; (ncol,nlay)
-    #                                            plev,    # level pressures [Pa, mb]; (ncol,nlay+1)
-    #                                            tlay      # layer temperatures [K]; (ncol,nlay)
-    # real(FT), dimension(:),   intent(in   ) :: tsfc      # surface skin temperatures [K]; (ncol)
-    # type(ty_gas_concs),       intent(in   ) :: gas_desc  # Gas volume mixing ratios
-    # # output
-    # class(ty_optical_props_arry),
-    #                           intent(inout) :: optical_props # Optical properties
-    # class(ty_source_func_lw    ),
-    #                           intent(inout) :: sources       # Planck sources
-    # character(len=128)                      :: error_msg
-    # # Optional inputs
-    # real(FT), dimension(:,:),   intent(in   ),
-    #                        optional, target :: col_dry,   # Column dry amount; dim(ncol,nlay)
-    #                                            tlev        # level temperatures [K]; (ncol,nlay+1)
+                       tlev=nothing)
+
+Compute gas optical depth and Planck source functions,
+given temperature, pressure, and composition
+
+Inputs
+ - this (`ty_gas_optics_rrtmgp` structure)
```

Comment:
```suggestion
 - `this` (`ty_gas_optics_rrtmgp` structure)
```
Based on [other documentation in clima](https://climate-machine.github.io/CLIMA/latest/Common/MoistThermodynamics.html#CLIMA.MoistThermodynamics.air_density), we should probably wrap all function arguments in backticks.